pkg/controller/cloudwatchmetrics/cloudwatchmetrics_controller.go

```diff
@@ -0,0 +1,111 @@
+package cloudwatchmetrics
```

Comment:
Can we rename this to cloud resource metrics, or cloud metrics. To allow for it to be provider-independent.

We'll likely end up using this for GCP and Azure too.

Reply:
Will do